lib/cesiumGeometryToGltfPrimitive.js

```diff
@@ -1,10 +1,74 @@
 'use strict';
+var Cesium = require('cesium');
+var findAccessorMinMax = require('./findAccessorMinMax');
+var getUniqueId = require('./getUniqueId');
+var mergeBuffers = require('./mergeBuffers');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+var uninterleaveAndPackBuffers = require('./uninterleaveAndPackBuffers');
 var writeAccessor = require('./writeAccessor');
 
+var WebGLConstants = Cesium.WebGLConstants;
+
 module.exports = cesiumGeometryToGltfPrimitive;
 
+// Creates a buffer location to write the new values to. mapGeometryAttributeToPrimitive does the actual writing
+function createAttributeSemantic(gltf, primitive, semantic, packedLength) {
+    var type;
+    switch (semantic.replace(/([0-9]|_)/g, '')) {
```

Comment:
@lilleyse Can have the final verdict on this one, but I'm pretty sure I remember saying not to do this in an earlier revision.

It's not the worst thing in the world, but you do make a new string every single time this statement executes which is a little wasteful.

This should probably be more like:
```javascript
if (semantic.indexOf('POSITION') === 0 || semantic.indexOf('NORMAL') === 0) {
    type = 'VEC3';
} else if (semantic.indexOf('TEXCOORD') === 0) {
    type = 'VEC2';
}
```